lib/ansible/modules/cloud/amazon/ec2_scaling_policy.py

```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/python
-# Copyright: Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-from __future__ import absolute_import, division, print_function
-__metaclass__ = type
-
+# This file is part of Ansible
```

Comment:
Isn't the current copyright right?

Reply:
Umm, it does look that way doesn't it. Again, an old change before the copyright standard was locked down, at least in my head!

Reply:
Ah, I know what happened - the copyright and `from __future__` stuff got changed well after this PR was made in  9c81257c